backend/protocol_rpc/endpoints.py

```diff
@@ -255,25 +255,23 @@ def get_transaction_by_id(
 
 def get_contract_state(
     accounts_manager: AccountsManager,
-    msg_handler: MessageHandler,
     contract_address: str,
     method_name: str,
     method_args: list,
 ) -> dict:
-    if not accounts_manager.is_valid_address(contract_address):
+    if not address_is_in_correct_format(contract_address):
```

Comment:
@AgustinRamiroDiaz where is this imported?

Reply:
fixed!